spec.bs

```diff
@@ -3291,6 +3310,56 @@ at the expense of some utility.
   </wpt>
 </div>
 
+<h3 id=credentialless-monkeypatch>Iframe credentialless</h3>
+
+The [[!IFRAME-CREDENTIALLESS]] specification defines a new object, the
+[=page credentialless nonce=]. At a high level, the
+[=fenced frame config instance/partition nonce=] serves the same purpose as the
+[=page credentialless nonce=] (partitioning storage and network), but scoped to each fenced
+frame rather than to each credentialless iframe. The
+[=fenced frame config instance/partition nonce=] is also used to revoke network access in fenced
+frames. Perform the following monkeypatches to the [[!IFRAME-CREDENTIALLESS]] spec.
+
+Add the following algorithm:
+
+<div algorithm>
+  To <dfn for="browsing context">compute the effective partition nonce</dfn> given a
+  boolean |credentialless| and [=fenced frame config instance/partition nonce=] or null
+  |newFencedFrameNonce|:
+
+  1. If |credentialless| is true, return |this|'s [=top-level browsing context=]'s
+     [=page credentialless nonce=].
+
+  1. If |newFencedFrameNonce| is not null, return |newFencedFrameNonce|.
+
+  1. Let |instance| be |this|'s [=browsing context/fenced frame config instance=].
+
+  1. If |instance| is null, return null.
```

Comment:
```suggestion
  1. If |instance| is null, then return null.
```

Per convention.

Reply:
This line was removed when consolidating with the line below it, per your previous comment.